webview/src/setup/components/dvc/DvcEnvDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { ReactElement } from 'react'
+import { DvcCliDetails } from 'dvc/src/setup/webview/contract'
+import { MAX_CLI_VERSION, MIN_CLI_VERSION } from 'dvc/src/cli/dvc/contract'
+import styles from './styles.module.scss'
+import { selectPythonInterpreter, setupWorkspace } from '../messages'
+
+const InfoRow: React.FC<{
+  title: string
+  text: string | ReactElement
+}> = ({ title, text }) => (
+  <tr>
+    <td className={styles.envDetailsKey}>{title}</td>
+    <td className={styles.envDetailsValue}>{text}</td>
+  </tr>
+)
+
+interface DvcEnvDetailsProps extends DvcCliDetails {
+  isPythonExtensionInstalled: boolean
+}
+
+export const DvcEnvDetails: React.FC<DvcEnvDetailsProps> = ({
+  exampleCommand,
+  version,
+  isPythonExtensionInstalled
+}) => {
+  const versionText = `${
+    version || 'Not found'
+  } (required >= ${MIN_CLI_VERSION} and < ${MAX_CLI_VERSION}.0.0)`
+  const commandText = exampleCommand || 'Not found'
+  const command = (
+    <>
+      <span className={styles.command}>{commandText}</span>
+      <span className={styles.actions}>
+        <button className={styles.buttonAsLink} onClick={setupWorkspace}>
+          Configure
+        </button>
+        {isPythonExtensionInstalled && (
```

Comment:
[I] Bit of an edge case but if we are not using the python extension to "discover" the CLI then I don't think we want to display this as it can be confusing:


https://user-images.githubusercontent.com/37993418/234448871-d3b8ea43-dc30-4d0b-b110-f7beb9293104.mov

This is why trying to jam everything into a single field is hard.

Reply:
Good catch! I'll only show it if the user is using the python extension or, if that's difficult to check, remove it entirely.